new_project/examples/separate_apps/src/negotiation.ts

```diff
@@ -0,0 +1,64 @@
+import axios from "axios";
+import express from "express";
+
+export interface Order {
+    key: string;
+    id: string;
+    valid_until: number;
+    bid: {
+        ledger: string;
+        asset: string;
+        amount: string;
+        network: string;
+    };
+    ask: {
+        ledger: string;
+        asset: string;
+        amount: string;
+        network: string;
+    };
+    execution_params: {
```

Comment:
As said above: I don't think this should be part of the order. For me there is another level of abstraction in between: 
1. What I show to the taker (= order + price relevant parameters as part of the order)
2. What I pass to the execution (= order + execution params)

Reply:
I do not follow 🙃

The taker needs to receive all the details at once through the api provided by the maker and then extract the execution params from it and pass it to the execution phase.